server.js

```diff
@@ -33,8 +71,61 @@ app.post('/api/createUser', (req, res) => {
 	});
 
 	connection.end();
+
+})
+
+app.get('/api/getEvents', (req, res) => {
+
+	let connection = mysql.createConnection(config);
+	//will prob need to edit sql statement to account for search features later
```

Comment:
For now, I think this request is perfectly fine. We can add a future request (something like `searchEvents`) that returns a similar but filtered output.